doc/ikfast_tutorial.rst

```diff
@@ -38,26 +38,12 @@ OpenRAVE Installation
 ----------------------
 **Binary Install**
 
-OpenRave is available for Ubuntu up to version 12.04 using these commands::
+ sudo add-apt install ros-indigo-openrave
```

Comment:
> apt-get works only for indigo, meaning 14.04 http://rosindex.github.io/p/openrave/#indigo

No I meant - the command ``add-apt`` is new in 16.04 and not available on 14.04 - on my system I get: ``add-apt: command not found``

> at this moment, we release only indigo, so indigo is ok, not so difficult to release on jade, kinetic

ah i didn't realize that. can we work on releasing to jade and kinetic too?

> how about Binary package of OpenRave from ROS repository is available for Ubuntu version 14.04 using these commands

also specify not just 14.04 but indigo is required (not jade at the moment)

thanks!